src/screens/GalleryScreen.tsx

```diff
@@ -0,0 +1,21 @@
+import { StatusBar } from 'expo-status-bar';
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+
+export default function GalleryScreen() {
+  return (
+    <View style={styles.container}>
+      <Text>Welcome to Gallery Screen!</Text>
+      <StatusBar style="auto" />
```

Comment:
You don't have to put `<StatusBar/>` inside each screen. you can do it globally in `App.tsx`:
```tsx
export default function App() {
  return (
    <>
      <RootNavigator />
      <StatusBar style="auto" />
    </>
  );
}